styles/components/_personal-bio.scss

```diff
@@ -0,0 +1,35 @@
+.cmp-personal-bio {
+  $full-width: 100%;
+  $header-line-height: 39px;
+  $content-line-height: 29px;
+  $mobile-screen-width: 40rem;
+
+  letter-spacing: -0.02em;
+  width: 100%;
+
+  &__header {
+    font-family: font-family("sans-extended");
+    font-size: font-size("lg");
+    font-weight: 600;
+    line-height: calc($header-line-height / 2);
+    margin: 0;
+    margin-bottom: space("md");
```

Comment:
In mobile view, it looks like the margin spacing might be differing from the component (skills) above.

![Screen Shot 2022-05-19 at 10 59 04 AM](https://user-images.githubusercontent.com/49104801/169356563-69431745-fc8f-4722-b314-753bc2adf8d6.png)

Reply:
Same in Ipad ~